src/charm.py

```diff
@@ -320,27 +311,48 @@ def _reset_environment() -> None:
         os.mkdir(CONFIG_PATH)
         os.mkdir(SERVICE_PATH)
 
-    def _get_bind_address(self) -> Optional[str]:
-        """Returns bind address."""
-        return (
-            ip_from_iface(bind_address_subnet)
-            if (bind_address_subnet := self.model.config.get("bind-address-subnet"))
-            else ip_from_default_iface()
-        )
-
-    def _get_current_status(self) -> ActiveStatus:
-        """Returns current status."""
-        status_type = ActiveStatus
+    @property
+    def _active_status_msg(self) -> str:
+        """Returns msg of current status."""
         status_msg = ""
-        if self._stored.installed:
+        if self._get_peer_data("installed"):
             status_msg = "SW installed."
-        if self._stored.started and service_active(SRS_ENB_SERVICE):
+        if self._get_peer_data("started") and service_active(SRS_ENB_SERVICE):
             status_msg = "srsenb started. "
-            if mme_addr := self._stored.mme_addr:
+            if mme_addr := self._mme_addr:
                 status_msg += f"mme: {mme_addr}. "
-            if self._stored.ue_attached and service_active(SRS_UE_SERVICE):
+            if self._get_peer_data("ue_attached") and service_active(SRS_UE_SERVICE):
```

Comment:
Same for `self._get_peer_data("ue_attached")`

Reply:
Removed `peer_data` here